src/sentry/options/manager.py

```diff
@@ -144,9 +144,11 @@ def get(self, key, silent=False):
             if result is not None:
                 # HACK(mattrobenolt): SENTRY_URL_PREFIX must be kept in sync
                 # when reading values from the database. This should
-                # be replaced by a signal.
+                # be replaced by a signal. We also want to keep the SENTRY_OPTIONS["system.url-prefix"]
+                # be in sync with SENTRY_URL_PREFIX to avoid warnings
```

Comment:
```suggestion
                # be replaced by a signal. We also want to keep the SENTRY_OPTIONS["system.url-prefix"]
                # in sync with SENTRY_URL_PREFIX to avoid warnings
```

Typo.